test/FunctionalTests/Microsoft.OData.Core.Tests/Json/DefaultJsonWriterFromStreamFactoryTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="DefaultJsonWriterFromStreamFactoryTests.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+#if NETCOREAPP3_1_OR_GREATER
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Microsoft.OData.Json;
+using Xunit;
+
+namespace Microsoft.OData.Tests.Json
+{
+    public class DefaultJsonWriterFromStreamFactoryTests

```

Comment:
```suggestion
    public sealed class DefaultJsonWriterFromStreamFactoryTests
```

Reply:
It doesn't seem like we normally make test classes sealed. What's the value of doing this for test classes?

Reply:
I don't differentiate between test classes and _____ (whatever we want to call all of the other classes that exist). Good practice is good practice.

Reply:
I have sealed the classes. I guess the other classes are used in production and get consumed or extended by other libraries or applications, but tests don't.